src/main/java/br/com/alura/comex/processadores/Processador.java

```diff
@@ -0,0 +1,9 @@
+package br.com.alura.comex.processadores;
+
+import br.com.alura.comex.Pedido;
+
+import java.util.List;
+
+public interface Processador {
+    List<Pedido> imprimir() throws Exception;
```

Comment:
Acho que o nome do método não está muito bom. Algo como `processa` seria melhor:

```suggestion
    List<Pedido> processa() throws Exception;
```